src/main/scala/com/github/gvolpe/fs2/Resources.scala

```diff
@@ -44,31 +40,31 @@ object ResourcesApp extends Resources[IO]
   *
   * Finally, R3 was done showing an availability of one once again.
   * */
-class Resources[F[_]: Effect] extends StreamApp[F] {
+object ResourcesApp extends IOApp {
 
-  override def stream(args: List[String], requestShutdown: F[Unit]): fs2.Stream[F, ExitCode] =
-    Scheduler(corePoolSize = 4).flatMap { implicit scheduler =>
+  def stream[F[_]: ConcurrentEffect: Timer]: fs2.Stream[F, Unit] =
```

Comment:
Maybe over-constrained?

Reply:
I rewrote `ResourcesApp` and removed the streams